wetlab/models.py

```diff
@@ -1309,10 +1309,10 @@ def create_lib_pool(self, pool_data):
 class LibraryPool(models.Model):
     register_user = models.ForeignKey(User, on_delete=models.CASCADE)
     pool_state = models.ForeignKey(PoolStates, on_delete=models.CASCADE)
-    run_process_id = models.ForeignKey(
-        RunProcess, on_delete=models.CASCADE, null=True, blank=True
-    )
-
+    # To be deleted in release 3.2.0 or higher
+    # replace by run_process_id from foreign key to many to many
+    run_process = models.ManyToManyField(RunProcess, blank=True)
```

Comment:
Pero no habáimos dicho que no tiene que ser un many-to-many, sino 1 library - n runs?

Reply:
Si, habria que hacerlo bien, como tenia en mente hacer el many to many, lo puse asi, pero habria que hacerlo como dijimos

Reply:
Sí, esto déjalo bien, porque entiendo que tb afecta al script de migración, no tiene sentido hacerlo dos veces